telemetry/specs/INT.mdk

```diff
@@ -992,11 +1009,25 @@ each bit corresponds to a specific standard metadata as specified in Section 3.
   Details of the metadata semantics YANG model can be accessed at the link below:
   https://github.com/p4lang/p4-applications/blob/master/telemetry/code/models/p4-dtel-metadata-semantics.yang
   
-  Each instruction requests 4 bytes of metadata to be inserted at each hop,
-  except if bit 6 is set, which requires 8 bytes of metadata. Per-hop
-  metadata length is set accordingly at the INT source.
-  
+  Bits 0 - 13 are Baseline INT Instructions and Bit 14, Domain Specific Instruction.
+  Domain Specific Instruction is an instruction that requires additional 
+  processing of Domain Specific ID, Domain Specific Flags (DS Flags) and 
+  Domain Specific Field (DS Field). If the Domain Specific ID doesn't match the 
+  Domain ID of the node, then the transit node is required to pad the node's 
+  INT Metadata stack with the special all-ones reserved value for Domain Specific 
+  Metadata length as calculated by subtracting the Baseline Metadata Length from Hop_ML.
   
+  If the Domain Specific ID matches the node's Domain ID then additional processing 
+  of the Domain Specific Flags and Domain Specific Field is required and 
+  Domain Specific Metadata is appended to the Baseline Metadata before 
+  Checksum Complement is inserted.
+
+  Each instruction requests 4 bytes of metadata to be inserted at each hop,
+  except if bit 6 and bit 14 is set. If bit 6 is set, the instruction requires 
```

Comment:
s/"bit 6 and bit 14"/"bit 6 or bit 14"
or "bit 6 and/or bit 14"

Reply:
Fixed it. Bit 14 is no longer needed.